ggml/src/ggml-backend.cpp

```diff
@@ -584,11 +588,15 @@ struct ggml_backend_registry {
 #ifdef GGML_USE_RPC
         register_backend(ggml_backend_rpc_reg());
 #endif
+
 #ifdef GGML_USE_AMX
         register_backend(ggml_backend_amx_reg());
 #endif
 
-        // TODO: kompute, cann
+#ifdef GGML_USE_CANN
+        register_backend(ggml_backend_cann_reg());
+#endif
```

Comment:
Do not add extra lines, use the same format as the rest of the backends.

Reply:
Thanks for your review, I have removed the blank lines and the formatting is consistent with the rest of the backends. Function is normal after removing.